springwolf-examples/springwolf-kafka-example/src/main/java/io/github/stavshamir/springwolf/example/kafka/dtos/XmlPayloadDto.java

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0
+package io.github.stavshamir.springwolf.example.kafka.dtos;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class XmlPayloadDto {
```

Comment:
Is this dto required, when there are no xml specific annotation required?

Reply:
Lets discuss if we should add Annotation to demonstrate that they work (and which annotations work)